portfolio/src/main/webapp/script.js

```diff
@@ -42,7 +42,10 @@ function getMessage() {
   fetch('/data').then(response => response.json()).then((messages) => {
     console.log(messages);
     let html = '';
-    let numDisplay = getParameter("numComments", 10);
```

Comment:
Just curious as to why this doesn't work?  I would have thought that even if a button wasn't clicked this call would have returned 10?

Reply:
Apparently it didn't. At least through my testing, whenever I loaded the page, there would be no comments displayed since there's no query string. Thus, the getParameter function returns null, causing numDisplay to basically become insignificant in the following loop. Whenever you click on one of the comment buttons, the query string gets updated to contain the desired parameter, meaning that the correct number of comments gets displayed on the website. Thus, I had to account for when there's no query string, and make a default value accordingly. I hope this makes sense.